### DISCUSSION TITLE
Error Handling

### DISCUSSION THREAD

**Discussion by SebastianFernandez893** [ranaroussi/yfinance#1555] [Q&A] [answered]
Hi I'm having some issues with error handling, that comes from a lack of experience with Python. 

```
def find_Max(company):
    try:
        hist = company.history(period="5y")
        print(company)
        maximum = max([x for x in hist["Close"]])
    except:
        print("ERROR")
    
```

find_Max takes the Ticker object and calculates the five year max from the dataframe.
I've imported a list of S&P500 tickers, but someof them has been delisted. I'm trying to get the program to skip over that company if an error is raised in this block of code which can be found at base.py lines 156-168.

```
if tz is None:
                # Every valid ticker has a timezone. Missing = problem
                err_msg = "No timezone found, symbol may be delisted"
                shared._DFS[self.ticker] = utils.empty_df()
                shared._ERRORS[self.ticker] = err_msg
                if raise_errors:
                    raise Exception('%s: %s' % (self.ticker, err_msg))
                else:
                    logger.error('%s: %s' % (self.ticker, err_msg))
                return utils.empty_df()
```

The error from yfinance pops up
`ERROR ANTM: No data found, symbol may be delisted`
as well as the exception for the arguments inside max being empty.
I don't really know how to get that exception to be passed into my own lines of code. I'd appreciate any help and please let me know if this is an incorrect spot for this and if it'd be better suited to post this on StackOverflow.

**Comment by ValueRaider** [comment, marked as answer]
Are you aware the API is documented in the Wiki? This will answer your question.

If you weren't, please suggest how to improve awareness.

**Reply by SebastianFernandez893** [reply]
I was not aware of that. Part of that could be because of my inexperience, but I'd suggest making a subsection under Quick Start saying something like 
# For more information on APIs please check the wiki

At the moment it's just a footnote right before logging and pretty easy to miss.
Checking the wiki did help though, thank you!

**Reply by ValueRaider** [reply]
Finally getting around to addressing this. Can you review this new README? https://github.com/ranaroussi/yfinance/tree/improve-readme#multiple-tickers

EDIT: Feedback here: #1586
